share/instruments/instruments.xml

```diff
@@ -11045,13 +11005,10 @@
                   <description>Bass Guitar (Tablature)</description>
                   <musicXMLid>pluck.bass</musicXMLid>
                   <stafftype staffTypePreset="tab4StrCommon">tablature</stafftype>
-                  <!--                  <clef>TAB</clef> -->
                   <genre>common</genre>
                   <genre>popular</genre>
             </Instrument>
             <Instrument id="acoustic-bass">
-                  <!-- Same instrument as the orchestral contrabass, but used by default plucked, rather than bowed
-                  can't init from it, as it has not been read yet. -->
```

Comment:
a way to keep such comments might be usefull

Reply:
It has now been retained a couple of lines down.